src/main/java/jp/ac/keio/bio/fun/xitosbml/cli/CliRun.java

```diff
@@ -30,6 +30,8 @@
 	String inputValue;
 	@Option(names = "-o", required = true, description = "The path to output XML file if input is image file")
 	String outputValue;
+	@Option(names = "-ds", description = "To show the domain hierarchy structure. False by default.")
```

Comment:
It is customary to define an option with a single hyphen with a single letter and double hyphen with multiple letters. For example, the help option is defined as `-h` and `--help`.

So we should follow that convention.

Reply:
Ok. Will do so.